Makefile

```diff
@@ -24,6 +24,19 @@ BOOTSTRAP_IMAGE ?= ghcr.io/cscetbon/casskop-bootstrap:0.1.9
 TELEPRESENCE_REGISTRY ?= datawire
 KUBESQUASH_REGISTRY:=
 
+
+VERSION ?= 2.1.5
```

Comment:
let's grab this from `version/version.go`. Don't want to have to modify all files every time there is a new version